### ISSUE TITLE
Feature request:  Archive Series

### ISSUE THREAD

**Issue by xios01** [Sonarr/Sonarr#4973]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

At present, Sonarr only offeres an option to “Delete” a TV series.  

However, if I want to keep an old or ended TV Series, but no longer want that series (monitored or unmonitored) under Sonarr’s management unnecessarily.  I presently have to copy/move that series to my “Archive TV” library folder manually.  (I keep “Archive TV” as a seperate defined  Library in Plex.) then, once the move is complete, I then have to delete the series from Sonarr.

No the manual process is not difficult, but rather time consuming and inconvient… 

### Describe the solution you'd like

when editing a TV series that you want to keep for your media center, but no longer want in Sonarr.  Sonarr’s current only option is to “Delete” the series.  I propose that next to the “delete” is an option called “Archive Series”.

- Under settings, you define an Archive Series folder that is outside of Sonarr’s management (maybe the copy vs move/hardlinking options as well)
- When the “Archive Series” button is pressed, the series, its related seaons and episodes are moved to the “Archive” defined folder
- Sonarr should probably notify plex that the series has been removed from the main folder so that it can update the libraries.
- finally Sonarr would proceed with removing the series from Sonarr as if it had been deleted.

### Describe alternatives you've considered

have not considered alternatives outside of my own manual method, which isn’t difficult, but can be a little more time consuming and inconvient.

### Anything else?

Not at this time…

**Comment by markus101** [comment]
No plans for an achieving feature, but you can achieve this in Sonarr in a few steps:

1. Edit the series and change the path (or change the root folder)
2. Tell Sonarr to move the files
3. Remove the series from Sonarr

**Comment by CLHatch** [comment]
> No plans for an achieving feature, but you can achieve this in Sonarr in a few steps:
> 
> 1. Edit the series and change the path (or change the root folder)
> 2. Tell Sonarr to move the files
> 3. Remove the series from Sonarr

Another option that might work better is to change `3. Remove the series from Sonarr` to `3. Unmonitor the series in Sonarr`.  This way, he won't end up downloading the same show again accidentally, it is seen to be "Archived" because of the root folder location.  Could also add an "Archived" tag, possibly.  It would also then be easy to "unarchive" it by changing the root folder back.

**Comment by Dnkhatri** [comment]
I have been think about a similar option as well. Currently I have 600+ shows with over 14k Episodes but less then 20 shows are monitored. The rest are unmonitored as they are asian single season shows and have ended. I could take them out of sonarr  but then they will no longer be available in bazaar etc. 
        Currenly sonarr refreshes these series daily on schedule and I think that is wasteful. An unmonitored and archived option would mean  sonarr will no longer scan those series for updates or contack skyhook for the data. I dont care about moving the show like the op out of sonarr but do think a setting where a show is not updated or its folders scanned when unmonitored and archived should be an option

**Comment by bakerboy448** [comment]
Duplicate of #315

There's no plans for this as Markus said.
